python/server/ServerWrapper.cpp

```diff
@@ -475,18 +524,8 @@ namespace {
         int SpawnLimit()
         { return m_monster_fleet_plan->SpawnLimit(); }
 
-        bool Location(int object_id) {
-            // get the universe object to test and check if it exists
-            TemporaryPtr<UniverseObject> obj = GetUniverseObject(object_id);
-            if (!obj) {
-                ErrorLogger() << "MonsterFleetPlanWrapper::Location: Couldn't get object with ID " << object_id;
-                return false;
-            }
-
-            // get location condition and evaluate it with the specified universe object
-            // if no location condition has been defined, any object matches
-            const Condition::ConditionBase* location_test = m_monster_fleet_plan->Location();
-            return (!location_test || location_test->Eval(obj));
+        list Locations(list systems) {
+            return FilterIDsWithCondition(m_monster_fleet_plan->Location(), systems);
```

Comment:
Ok, I wasn't aware that checking all systems at once with `Locations` is almost as fast as checking only one with `Location`. So, that means a line like this:
```
if system in monster_fleet.Locations():
    ...
```
is so close to check `system` with a call to `Location` performance-wise that it's practically pointless to keep the `Location` function, right? In this case I agree.